### DISCUSSION TITLE
Artnet and simple power plug

### DISCUSSION THREAD

**Discussion by 1ucay** [arendst/Tasmota#19833] [Q&A]
Hi, it is possible to control power on / off over Artnet?
I would like to control this plug
https://templates.blakadder.com/athom_PG01V3-EU16A.html

Thank you.

**Comment by s-hadinger** [comment]
I'm afraid not. It's maybe possible via DDP though 

What is your setup?

**Reply by s-hadinger** [reply]
Since it's esp32c3 it's still doable with Berry scripting

**Comment by 1ucay** [comment]
I don't have a plug at the moment.. So it is possbile somehow?

**Reply by s-hadinger** [reply]
I would say that anything is possible with some time and development skills ;)

Right now, it's not possible out of the box. Can we explore alternative options ? (DDP, Web, MQTT)
